src/index.html

```diff
@@ -5,9 +5,54 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
-    <link rel="stylesheet" href="./css/style.css">
-
     <title>Hakim Portfolio</title>
+
+    <!-- Global styles -->
+    <link rel="stylesheet" href="./css/reset.css" />
+    <link rel="stylesheet" href="./css/style.css" />
+    <link rel="stylesheet" href="./css/home.css" />
+    <!-- components -->
+    <link rel="stylesheet" href="./css/components/toolbar.css" />
+    <link rel="stylesheet" href="./css/components/headline.css" />
+    <!-- google fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body></body>
+  <body>
+    <header class="header">
+      <nav class="toolbar">
+        <div class="toolbar__logo-wrapper">
+          <span class="toolbar__logo">Hakim</span>
+        </div>
+        <button class="toolbar__menu-button" type="button">
+          <img src="./assets/icons/menu.svg" alt="Menu icon" />
+        </button>
+      </nav>
```

Comment:
* You did a very great job with the UI 
  - [ ]  As mentioned in the requirements, the logo should be wrapped in a < `a` > tag that redirects to the home page(in this case its the same page).
  - [ ] The logo should be out of the `nav`, but should still be in the header. Please kindly fix that.

Reply:
@m05e5 thank you for your comment, please I would like to know where is the requirement that says logo should be an anchor (`<a>`) tag?